src/components/EnterIssue.svelte

```diff
@@ -0,0 +1,62 @@
+<script>
+  import { getIssues } from "../helpers/oktokit";
+  import Button from "../ui/Button/Button.svelte";
+  import TextField from "../ui/TextField/TextField.svelte";
+  import Heading from "../ui/Heading/Heading.svelte";
+
+  let issueId;
+  let promise = Promise.resolve("");
```

Comment:
This is an interesting pattern. Is it common in the Svelte world?

Reply:
So I've used this together with svelte's [await blocks](https://svelte.dev/examples#await-blocks):

```js
{#await promise}
	...loading
{:then success}
	 {success}
{:catch error}
	{ error.message}
{/await}
``` 
In the example in the docs a call is made right away, but in our case only after a button is pressed. So I've used a fake promise resolve for the initial state. Not sure whether it's common. There is another way of doing it: https://svelte.dev/repl/b9fc662a253443dc901ff189ce1cdd4b?version=3.16.7 (this might be cleaner actually)